docs/conf/eu/2016/news/schedule.md

```diff
@@ -0,0 +1,85 @@
+
+We're putting the finishing touches on the conference, so this email
+contains more information for you about this year's events. We also hope you'll be able to join most if not all of the events outside the main conference talks, because they're some of our favorite parts of the event.
+
+Conference Schedule
+-------------------
+
+The [schedule][schedule] for the conference is now fully filled out.
+We're excited about the talks, Writing Day,
+and all the other wonderful festivities we have every year.
+
+[schedule]: http://www.writethedocs.org/conf/eu/2016/schedule/
+
+
+Saturday - Boat Trip
+----------------------
+
+We're planning a scenic boat tour on Saturday afternoon. Attendance is included with your conference ticket, but please do confirm that you have lets us know that you're coming, so we can book enough boats!
+
+
+Sunday - Writing Day and Conference Reception
+---------------------------------------------
+
+During the day on Sunday, at the [conference venue][autoklub-map], we'll have our [Writing Day][writing-day], followed by the reception in the evening.
+
+The Writing Day is modeled after the concept of ["sprints"][sprints] from
+the open source and software development world. The idea is simple: to get a
+bunch of interesting people in a room together and have them work towards shared
+goals.
+
+We encourage everyone to drop by on Sunday evening for our conference reception.
+You'll have a chance to get acquainted with each other over some drinks and
+snacks, plus it's a great place to find some folks to grab dinner with,
+so you can continue your conversations over more substantial food.
+
+[writing-day]: http://www.writethedocs.org/conf/eu/2016/writingday/
+[autoklub-map]: https://goo.gl/maps/w3gGKECb3HQ2
+[sprints]: http://en.wikipedia.org/wiki/Sprint_%28software_development%29
+
+
+
+Conference Party
+----------------
+
+This year we'll be holding the conference party at last years conference venue, the awesome [Klub Lavka][lavka-map]! Wind down and talk about the day’s talks in a relaxed and friendly environment. Free drinks of all varieties, snacks, and sweets will be provided.
+
+[lavka-map]: https://goo.gl/maps/TgWcR2DAwWt
+
+Monday and Tuesday - Conference
+-------------------------------
+
+We've released the breakdown of [our daily schedule][daily-schedule].
+The doors open at 10:00 on both days,
+so that you can arrive early, get yourself settled and grab some breakfast. The conference begins at 10:00. Talks
+run all day, until shortly after 17:00.
+
+[daily-schedule]: http://www.writethedocs.org/conf/eu/2016/schedule/#monday-september-19th-day-1
+
+----
+
+This year will be the best Write the Docs yet!
+We have taken everyone's feedback and continued to tweak things to make them even better than
+before.
+As always, let us know if there is anything else we can improve.
+
+
+[email]: mailto:conf@writethedocs.org
+
+Sponsors
+---------
+
+Thanks to our latest sponsors:
```

Comment:
Should do a bit more here. "The conference wouldn't be possible without them" or somesuch.